Assets/BossRoom/Scripts/Shared/Net/UnityServices/Lobbies/LobbyContentHeartbeat.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using BossRoom.Scripts.Shared.Infrastructure;
+using BossRoom.Scripts.Shared.Net.UnityServices.Infrastructure;
+
+namespace BossRoom.Scripts.Shared.Net.UnityServices.Lobbies
+{
+    /// <summary>
+    /// Keep updated on changes to a joined lobby, at a speed compliant with Lobby's rate limiting.
+    /// </summary>
+    public class LobbyContentHeartbeat

```

Comment:
> Keep updated on changes to a joined lobby, at a speed compliant with Lobby's rate limiting.

Oh wait, that's not a heartbeat then, that's just polling. Very different from heartbeats which are used to manage timeouts for example.

Reply:
It actually does what is normally known as heartbeat, if you're host:

```
   if (m_localUser.IsHost)
            {
                m_LobbyAsyncRequests.DoLobbyHeartbeat(dt);
            }
```